LIB386/LIB_MENU/LIB_MENU.H

```diff
@@ -1,7 +1,7 @@
 #ifdef	WATCOM9
-#pragma	library ("f:\projet\lib386\lib_menu\lb9_menu.lib");
+#pragma	library ("c:\lba\lib386\lib_menu\lb9_menu.lib");
 #else
-#pragma	library ("f:\projet\lib386\lib_menu\lib_menu.lib");
+#pragma	library ("c:\lba\lib386\lib_menu\lib_menu.lib");
```

Comment:
Is there a way to relative set the path?

Reply:
You could remove those statements, as long as you pass the lib files as parameters to the linking program